src/components/Search/SearchPageHeader.tsx

```diff
@@ -43,35 +57,63 @@ function SearchPageHeader({query, selectedItems = {}, hash, clearSelectedItems,
         finished: {icon: Illustrations.CheckmarkCircle, title: translate('common.finished')},
     };
 
-    const selectedItemsKeys = Object.keys(selectedItems ?? []);
+    const selectedTransactionsKeys = Object.keys(selectedTransactions ?? []);
 
     const headerButtonsOptions = useMemo(() => {
-        const options: Array<DropdownOption<SearchHeaderOptionValue>> = [];
-
-        if (selectedItemsKeys.length === 0) {
-            return options;
+        if (selectedTransactionsKeys.length === 0) {
+            return [];
         }
 
-        const itemsToDelete = Object.keys(selectedItems ?? {}).filter((id) => selectedItems[id].canDelete);
+        const options: Array<DropdownOption<SearchHeaderOptionValue>> = [
```

Comment:
Since we are doing the native elsewhere, lets add the download option only if we are on web for now.